R/NielsonKuparuk.R

```diff
@@ -0,0 +1,278 @@
+  }
+}
+
+Kup9Q <- Kup9Q[-delete,]
+
+
+files <- list(IC_WeirQ, ICKUP_Q, Kup2AQ, Kup2TQ, Kup3BQ, Kup3TQ, Kup4Q, Kup5Q, Kup8Q, Kup9Q)
+
+#Separate date and time in order to edit
+
+for(i in 1:length(files)){
+  files[[i]] <- separate(data = files[[i]], col = Date_adt, into = c("Date", "Time"), sep = " ")
+}
+
+##Add ":00" at end of time
+for(i in 1:length(files)){
+  for(j in 1:length(files[[i]]$Time)){
+    files[[i]]$Time[j] <- paste(files[[i]]$Time[j], ":00", sep = "")
+  }
+}
+
+##Make hour two digits
+for(i in 1:length(files)){
+  for(j in 1:length(files[[i]]$Time)){
+    pattern = "^\\d{2}[[:punct:]]"
+    results <- grepl(pattern, files[[i]]$Time)
+    if(results[j] == FALSE){
+      files[[i]]$Time[j] <- paste("0", files[[i]]$Time[j], sep = "")
+    }
+  }
+}
+
+#Change date format
+files[[1]]$Date <- as.Date(files[[1]]$Date, format = "%m/%d/%Y")
+for(i in 1:length(files[[1]]$Date)){
+if(isTRUE(files[[1]]$Date[i] < as.Date("2000-01-01"))){
+  files[[1]]$Date[i] <- format(files[[1]]$Date[i], "20%y-%m-%d")
+}else{
+  files[[1]]$Date[i] <- format(files[[1]]$Date[i])
+}
+}
+
+for(i in 2:length(files)){
+    files[[i]]$Date <- as.Date(files[[i]]$Date, format = "%m/%d/%Y")
+  }
+
+#Combine date and time columns
+for(i in 1:length(files)){
+  files[[i]] <- unite(files[[i]], Date_adt, c(Date, Time), sep = " ")
+}
+
+#Format Date_adt
+for(i in 1:length(files)){
+  format(as.POSIXct(files[[i]]$Date_adt, tz = ""), format = "%Y-%m-%d %H:%M:%S")
+}
+
+#Data file editing complete!
+
+#Reassign edited files to new file names
+write.csv(files[[1]], file = "/home/mnguyen/Neilson/Neilson_IC_WeirQ.csv", row.names = FALSE) 
+write.csv(files[[2]], file = "/home/mnguyen/Neilson/Neilson_ICKUP_Q.csv", row.names = FALSE) 
+write.csv(files[[3]], file = "/home/mnguyen/Neilson/Neilson_Kup2AQ.csv", row.names = FALSE) 
+write.csv(files[[4]], file = "/home/mnguyen/Neilson/Neilson_Kup2TQ.csv", row.names = FALSE) 
+write.csv(files[[5]], file = "/home/mnguyen/Neilson/Neilson_Kup3BQ.csv", row.names = FALSE) 
+write.csv(files[[6]], file = "/home/mnguyen/Neilson/Neilson_Kup3TQ.csv", row.names = FALSE) 
+write.csv(files[[7]], file = "/home/mnguyen/Neilson/Neilson_Kup4Q.csv", row.names = FALSE) 
+write.csv(files[[8]], file = "/home/mnguyen/Neilson/Neilson_Kup5Q.csv", row.names = FALSE) 
+write.csv(files[[9]], file = "/home/mnguyen/Neilson/Neilson_Kup8Q.csv", row.names = FALSE) 
+write.csv(files[[10]], file = "/home/mnguyen/Neilson/Neilson_Kup9Q.csv", row.names = FALSE) 
+
+
+#Check if length and values the same
+test <- read.csv("/home/mnguyen/Neilson/Neilson_Kup9Q.csv", header=TRUE)
+test$Discharge_cms == Kup9Q$Discharge_cms
+length(test$Discharge_cms) == length(Kup9Q$Discharge_cms)
+#Good to go
+
+#EML
+
+eml <- read_eml("/home/mnguyen/Neilson/NeilsonMetadata.xml")
+
+#create attributes table
+
+library(RCurl)
+script <- getURL("https://raw.githubusercontent.com/NCEAS/datamgmt/master/R/create_attributes_table.R", ssl.verifypeer = FALSE)
+eval(parse(text = script))
```

Comment:
If you're still doing this, you can switch to this:

`devtools::install_github("NCEAS/datamgmt")` - only need to do this once
`library(datamgmt)`